link/src/main/java/com/stripe/android/link/analytics/DefaultLinkEventsReporter.kt

```diff
@@ -41,15 +42,19 @@ internal class DefaultLinkEventsReporter @Inject constructor(
     }
 
     override fun onSignupFailure(isInline: Boolean, error: Throwable) {
-        val params = mapOf(FIELD_ERROR_MESSAGE to error.safeAnalyticsMessage)
+        if (error.canLog) {
```

Comment:
I think we really want to log everything, it's better to get an event with an unknown, than no event at all.